src/tetromino/tetromino.py

```diff
@@ -105,6 +105,11 @@ def rotate_ccw(self):
                 Point((int(new_point[0]), int(new_point[1]))))
         self.state = self.state.prev()
 
+    def reset_position(self):
+        """Resets the tetromino to its original spawn position"""
+        self.origin = Point((SPAWN[self.id][0], SPAWN[self.id][1]))
+        self.sqrs = self.populate_sqrs()
```

Comment:
Should be squares now right?

Reply:
yea but i think we should merge #27 first, otherwise its gonna be weird lol